### PR TITLE
Add test cases for MySQLDialectExceptionMapper and PostgreSQLDialectExceptionMapper 

### LINKED ISSUE #20272: Add test cases for MySQLDialectExceptionMapper and PostgreSQLDialectExceptionMapper

### PR DESCRIPTION
**Pull request by sunkai-cai**
Fixes #20272.

### CODE REVIEW

File: shardingsphere-dialect-exception/shardingsphere-mysql-dialect-exception/src/test/java/org/apache/shardingsphere/dialect/mysql/mapper/MySQLDialectExceptionMapperTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+
+public class MySQLDialectExceptionMapperTest {
```
Comment (yx9o) [inline shardingsphere-dialect-exception/shardingsphere-mysql-dialect-exception/src/test/java/org/apache/shardingsphere/dialect/mysql/mapper/MySQLDialectExceptionMapperTest.java:40]: Please add final.

File: shardingsphere-dialect-exception/shardingsphere-postgresql-dialect-exception/src/test/java/org/apache/shardingsphere/dialect/postgresql/mapper/PostgreSQLDialectExceptionMapperTest.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import org.postgresql.util.PSQLState;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+
+public class PostgreSQLDialectExceptionMapperTest {
```
Comment (yx9o) [inline shardingsphere-dialect-exception/shardingsphere-postgresql-dialect-exception/src/test/java/org/apache/shardingsphere/dialect/postgresql/mapper/PostgreSQLDialectExceptionMapperTest.java:36]: Please add final.

File: shardingsphere-dialect-exception/shardingsphere-mysql-dialect-exception/src/test/java/org/apache/shardingsphere/dialect/mysql/mapper/MySQLDialectExceptionMapperTest.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    
+    private Collection<Object[]> getConvertParameters() {
+        return Arrays.asList(new Object[][]{
```
Comment (yx9o) [inline shardingsphere-dialect-exception/shardingsphere-mysql-dialect-exception/src/test/java/org/apache/shardingsphere/dialect/mysql/mapper/MySQLDialectExceptionMapperTest.java:43]: How about using Map here?

Comment (sunkai-cai) [reply shardingsphere-dialect-exception/shardingsphere-mysql-dialect-exception/src/test/java/org/apache/shardingsphere/dialect/mysql/mapper/MySQLDialectExceptionMapperTest.java:43]: this is a Two Dimensional Array， the first column is parameters serial.  Map maybe not suitable.

File: shardingsphere-dialect-exception/shardingsphere-mysql-dialect-exception/src/test/java/org/apache/shardingsphere/dialect/mysql/mapper/MySQLDialectExceptionMapperTest.java (hunk 1 above)
Comment (sunkai-cai) [reply shardingsphere-dialect-exception/shardingsphere-mysql-dialect-exception/src/test/java/org/apache/shardingsphere/dialect/mysql/mapper/MySQLDialectExceptionMapperTest.java]: done.

File: shardingsphere-dialect-exception/shardingsphere-postgresql-dialect-exception/src/test/java/org/apache/shardingsphere/dialect/postgresql/mapper/PostgreSQLDialectExceptionMapperTest.java (hunk 2 above)
Comment (sunkai-cai) [reply shardingsphere-dialect-exception/shardingsphere-postgresql-dialect-exception/src/test/java/org/apache/shardingsphere/dialect/postgresql/mapper/PostgreSQLDialectExceptionMapperTest.java]: this is ok

**Review by yx9o — APPROVED** [review]

**Pull request merged**